### ISSUE TITLE
Schema change introduced parsing error 

### ISSUE THREAD

**Issue by josefkarasek** [openshift/cluster-logging-operator#24]
Doesn't tell where the error is in the code, unfortunately. 
```
ERRO[0000] error syncing key (openshift-logging/example): failed to decode json data with gvk(logging.openshift.io/v1alpha1, Kind=ClusterLogging): v1alpha1.ClusterLogging.Spec: v1alpha1.ClusterLoggingSpec.Collection: v1alpha1.CollectionSpec.LogCollection: v1alpha1.LogCollectionSpec.FluentdSpec: v1alpha1.FluentdSpec.NodeSelector: ReadMapCB: expect { or n, but found ", error found in #10 byte of ...|elector":"logging-in|..., bigger context ...|ion":{"logCollection":{"fluentd":{"nodeSelector":"logging-infra-fluentd=true"},"type":"fluentd"}},"c|... 
```

**Comment by ewolinetz** [comment]
This was caused by bad formatting in the cr.yaml file. I updated this to be fixed in #23

**Comment by josefkarasek** [comment]
Thanks!!
